pkgs/development/python-modules/examples/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pydantic_1,
+  pytestCheckHook,
+}:
+buildPythonPackage {
+  pname = "examples";
+  version = "0-unstable-2021-05-26";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "timothycrosley";
+    repo = "examples";
+    # no commits for 3 years and I don't know what revision
+    # uses version on PyPI so I am packaging last commit
+    rev = "2667ad2793c008d4f4c1c62851c99bf7d0d36290";
+    hash = "sha256-4u5SiM9QOMLYFG6thX5bu6Hw2n0MFUvJlkaiUw/CyA4=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = [
+    pydantic_1
+  ];
```

Comment:
We are unable to propagate such a versioned package due to PYTHONPATH conflicts.
If it is introduced as a dependency of some application, it should be overridden in that derivation. Otherwise we cannot accept it in nixpkgs.

> Only unversioned attributes (e.g. pydantic, but not pypdantic_1) can be included in dependencies, since due to PYTHONPATH limitations we can only ever support a single version for libraries without running into duplicate module name conflicts.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Removed, thanks!